data/mods/More_Survival_Tools/snippets.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "type": "snippet",
+    "category": "note",
+    "//": "Random graveyard epitaph from Nethack, and also referencing prehistoric hunter profession in the mod.",
```

Comment:
Very well then. Could not think of a replacement snippet, but four more seems acceptable.